staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto

```diff
@@ -320,6 +320,14 @@ message ListMeta {
   // More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#concurrency-control-and-consistency
   // +optional
   optional string resourceVersion = 2;
+
+  // continue may be set if the user set a limit on the number of items returned, and indicates that
+  // the server has more data available. The value is opaque and may be used to issue another request
+  // to the endpoint that served this list to retrieve the next set of available objects. Continuing a
+  // list may not be possible if the server configuration has changed or more than a few minutes have
```

Comment:
Is this few minutes somehow defined? I think the actual time that needs to elapse may be different.
So I wouldn't be that specific saying about minutes.

WDYT?

Reply:
I think the problem is, if we do not specify some reasonable expectation here API consumers will likely not have any expectation.  We should be clear that this may be changed per consumer, but setting a rough magnitude is important.

Continue depends on historical MVCC values, which depends on compaction interval.  It's arbitrarily set to 5 minutes today.  Rate of change in the cluster creates dead values which increases resident size on disk and in memory, and compaction clears those and also makes historical reads not possible.  In the future we may tune this up.

Reply:
But isn't it that it may significantly vary?
If you start you paginated LIST right before the compation, you will have the data available for just very small amount of time, right? Or am i missing something?

If so, this on average will be in the order of magnitude of minutes, but can be either 1s or <compaction interval>. So it's not hard guarantee.

Or am I completely missing something?

Reply:
Is compaction not rolling? I thought it compacted up to a fixed time in the past, not all events up to the present

Reply:
Yeah, compaction keeps x minutes of history

Reply:
Ahh ok. Sorry. I forgot about it.
I think you're right that we are compacting up to "previous time when we did compaction". So indeed, with compaction happening every 5 minutes, the history time when you LIST will still be valid should be something between 5 and 10 minutes.